project/Andymyyon/README.md

```diff
@@ -0,0 +1,80 @@
+### Project Name:
+**Dynamic Airspace Management System (DAMS)**
+
+### Basic Information
+- **GitHub Repository URL**: [https://github.com/Andymyyon/DAMS]
+- **About Statement**: "Optimizing airspace allocation for air traffic control
+through dynamic Voronoi diagram partitioning to ensure safety and efficiency in
+air traffic management."
+
+### Problem to Solve
+- **Field**: Aviation, Air Traffic Management
+- **Background**: Air Traffic Management (ATM) is essential for ensuring the
+safety and efficiency of aircraft operations within the global airspace system.
+Traditional ATM systems face challenges in balancing controller workloads,
+especially in areas of high traffic density or during peak operational times.
+Dynamic allocation of airspace can offer a solution by adjusting sector sizes
+and responsibilities in real-time to ensure an equitable distribution of
+workload among air traffic controllers.
+- **Physics/Mathematics**: This project will leverage computational geometry,
+specifically Voronoi diagrams, to partition airspace dynamically based on the
+geographic locations of Air Traffic Control (ATC) towers or centers and the
+positions of aircraft in flight.
+- **Algorithm/Numerical Method**: We will use Voronoi diagram generation
+algorithms for dynamic airspace partitioning, coupled with real-time data
+processing and load-balancing algorithms to assess and adjust the distribution
+of air traffic control workload.
+
+### Prospective Users
+- The primary users of DAMS will be air traffic controllers and ATC operations
+managers, who will utilize the system to monitor and manage the dynamic
+allocation of airspace sectors, thus enhancing their situational awareness and
+decision-making capabilities.
+- The system can also be beneficial for aviation safety analysts and airspace
+planners in optimizing airspace structures and workflows.
+
+### System Architecture
+- **Input**: The system will take real-time and historical air traffic data
+(including aircraft positions, velocities, and flight plans) and ATC capacity
+and location data as inputs.
+- **Operations**: DAMS will parse and process the input data, compute dynamic
+Voronoi diagrams for airspace partitioning, assess workloads, and allocate or
+reallocate airspace sectors accordingly.
+- **Output**: The system will produce updated airspace sector maps and workload
+distribution reports. Alerts for manual intervention or adjustment by human
+controllers may also be generated.
+- **Interface**: Instead of a web-based dashboard, DAMS will offer a command-line
```

Comment:
A web-based GUI means you have 3 languages involved: C++, Python, and JavaScript.  It is wise to avoid it unless you are very familiar with web-based GUI.